iModelJsNodeAddon/presentation/ECPresentationSerializer.cpp

```diff
@@ -1119,6 +1131,365 @@ rapidjson::Document IModelJsECPresentationSerializer::_AsJson(ContextR ctx, Node
     return json;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+static Utf8String GetECValueTypeName(PrimitiveType type)

```

Comment:
- function name seems wrong - it's not taking an `ECValue`
- we could return `Utf8CP` here. that would allow us to place it into json with `rapidjson::StringRef` rather than copying it into `rapidjson::Value`

Reply:
Renamed to `ParsePrimitiveType`.
Returning Utf8CP now and replaced the two usages from rapidjson::Value to rapidjson::StringRef.